security/services/security-service/src/main/java/ezbake/security/service/ThriftTokenJSONProvider.java

```diff
@@ -33,15 +37,33 @@
  */
 public class ThriftTokenJSONProvider implements TokenJSONProvider {
     private static final Logger LOGGER = LoggerFactory.getLogger(ThriftTokenJSONProvider.class);
+    private static final ObjectMapper MAPPER = new ObjectMapper();
 
     @Override
     public EzSecurityTokenJson getTokenJSON(EzSecurityToken token, PKeyCrypto signer)
             throws TException, UnsupportedEncodingException {
 
         EzSecurityTokenJson json = new EzSecurityTokenJson();
-        json.setJson(new String(new TSerializer(new TSimpleJSONProtocol.Factory()).serialize(token), "UTF-8"));
+        String jsonString = new String(new TSerializer(new TSimpleJSONProtocol.Factory()).serialize(token), "UTF-8");
+        
         json.setSignature("");
-        return json;
-    }
 
+        if (token.getValidity() == null) {
```

Comment:
prefer to use the thrift api isSetValidity instead of directly checking for null